src/main/java/org/opencord/kafka/integrations/AaaKafkaIntegration.java

```diff
@@ -89,7 +109,29 @@ protected void unbindAuthenticationService(AuthenticationService authenticationS
             log.warn("Trying to unbind AuthenticationService but it is already unbound");
         }
     }
+    protected void bindAuthenticationStatService(AuthenticationStatisticsService authenticationStatisticsService) {
+        log.info("bindAuthenticationStatService");
```

Comment:
Remove unnecessary log statements. Just 1 success or failure log statement should be generated by this method. Same applies for next unbindXXXXX() method